Documentation: restructure and add 'how to' documentation

jfy133 opened the pull request:
My idea is that `tutorials` will be video screencast for `filter`

Currenty `convert` docs not written.

Partly closes #42

maxibor commented:
Can you integrate https://github.com/SPAAM-community/AMDirT/blob/dev/docs/source/filter.md in the filter how-to ?

jfy133 commented:
I was wondering abou that, but given the functionality should also hopefully present in `convert`, I wonder if we should have a standalone page for it... but i'll make sure to add (sorry I missed it first time around, it wasn't in the ToC in the rendered version so I kept forgetting it)

ilight1542 reviewed (changes requested):
Some updated wording, typo fixes, and will add new files (screenshots for how-to) directly to branch on new commit

jfy133 commented on docs/source/how_to/filter.md:
````diff
@@ -0,0 +1,108 @@
+bash ancientMetagenomeDir_curl_download_script.sh
+```
+
+and the sequencing files of the selected samples will be downloaded to the directory you run the script from
````
Good point! Will add

ilight1542 reviewed (commented):
I will update today with embedded youtube vids on relevant sections

jfy133 commented on docs/source/how_to/filter.md:
```diff
@@ -0,0 +1,120 @@
+
+### Loading the interface
+
+Before using `AMDirT filter` you will to make sure you have modern web-browser (Chrome, Firefox, Edge, Safari etc.) available on your machine. This will in most cases be your own laptop or desktop - not a server.
```
```suggestion
Before using `AMDirT filter` make sure you have modern web-browser (Chrome, Firefox, Edge, Safari etc.) available on your machine. This will in most cases be your own laptop or desktop - not a server.
```

jfy133 commented on docs/source/how_to/filter.md:
```diff
@@ -0,0 +1,120 @@
+
+![AMDirT side bar with download method menu](images/amdirt-filter-downloadmethod-select.png)
+
+Once you have selected a version and a table, the table itself should open in the empty pane.
```
```suggestion
Once you have selected a version and a table, the table itself should open in the empty pane.

> In most cases the most version is recommended, whereby you pick the most recent date e.g. v22.12 over v22.09 from December and September 2022 respectively
```

jfy133 commented:
@maxibor I think all the docs are technically now complete

(@ilight1542 I moved the videos rather to a dedicated `tutorials` sectoin and added one sentence descriptions of each video)

jfy133 commented:
Will need `autofill` docs too

maxibor commented on docs/source/how_to/filter.md:
```diff
@@ -0,0 +1,122 @@
+
+Once you have selected a version and a table, the table itself should open in the empty pane.
+
+> In most cases the most version is recommended, whereby you pick the most recent date e.g. v22.12 over v22.09 from December and September 2022 respectively
```
...most recent version...
```suggestion
> In most cases the most recent version is recommended, whereby you pick the most recent date e.g. v22.12 over v22.09 from December and September 2022 respectively
```

maxibor commented on docs/source/tutorials.md:
```diff
@@ -0,0 +1,47 @@
+
+## Loading AMDirT filter
+
+This tutorial shows how to activate a [pre-created](README#install) conda environment from a command-line interface, and load the graphical-user interface (GUI) in your browser.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EANpFh__6V8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
```
This step is only needed because AMDirT is not yet released. But loading a conda environment won't be necessary once it's released on pip/bioconda.

maxibor reviewed (commented):
Very nice work on the documentation !
Besides a few typos, and the comment on the conda env that won't be necessary once we release, it's ready for merging !
